C/[02]-variabel-dan-tipe-data.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Variabel dan Tipe Data
+created: 03-10-2022
+update: -
+---
+# 2. Variabel dan Tipe Data
+Pada bab ini akan membahas mengenai Variabel dan Tipe Data pada bahasa C.
+
+## Variabel
+- Variabel
+
+Variabel adalah wadah yang digunakan untuk menampung data/value. Bayangkan saja gelas dan susu. Analoginya gelas adalah tempat untuk menampung susu tersebut. 
+Jadi, dalam membuat program, tentu kita sangat membutuhkan variabel. Untuk mengenali variabel tersebut maka harus ada tipe data. 
+Jika kita memilih gelas maka gunakan gelas itu untuk menampung air, tidak mungkin kan gelas untuk menampung nasi. Oleh karena itu dalam implementasi ke dalam 
+bahasa pemrograman juga harus menggunakan tipe data sebagaimana harusnya.
```

Comment:
IMO, better kasih penjelasan secara komputerisasi, variable disimpen di mana? memory. jadi ga `out of nowhere` gitu

Tapi ini CMIIW, apakah perlu dijelasin se detail itu